display_information/display-annotation/README.md

```diff
@@ -0,0 +1,41 @@
+# Display annotation
+
+Display annotation from a feature service URL.
+
+![Image showing the Display Annotation sample](DisplayAnnotation.png)
+
+## Use case
+
+Annotation is useful for displaying text that you don't want to move or resize when the map is panned or zoomed (unlike labels which will move and resize). You can use annotation to place text at a fixed size, position, orientation, font, and so on. You may choose to do this for cartographic reasons or because the exact placement of the text is important.
```

Comment:
> for cartographic reasons or because the exact placement of the text is important.

A real example would help this use case.

Reply:
Very good point, I'll have a think and perhaps suggest an addition to the common readme.